packages/turf-clusters/README.md

```diff
@@ -0,0 +1,172 @@
+# @turf/clusters
+
+# getCluster
+
+Get Cluster
+
+**Parameters**
+
+-   `geojson` **[FeatureCollection](http://geojson.org/geojson-spec.html#feature-collection-objects)** GeoJSON Features
+-   `filter` **Any** Filter used on GeoJSON properties to get Cluster
```

Comment:
This type of questions would be better answered using `clusterEach` or `clusterReduce`.

Calculating how many total clusters you could use `clusterReduce` and `previousValue++` which would give you the total, or if you want to figure all of the values which created a bin you could also use `clusterReduce`.

> I'll add some more examples to `clusterReduce` & `clusterEach`